fiasco/ions.py

```diff
@@ -40,7 +40,7 @@ class Ion(IonBase, ContinuumBase):
         input formats.
     temperature : `~astropy.units.Quantity`
         Temperature array over which to evaluate temperature dependent quantities.
-    ioneq_filename : `str`, optional
+    ionization_fraction : `str`, optional
         Ionization equilibrium dataset
```

Comment:
```suggestion
    ionization_fraction : `str` or `float` or array-like, optional
        If a string is provided, use the appropriate "ioneq" dataset. If an array is provided, it must be the same shape
        as ``temperature``. If a scalar value is passed in, the ionization fraction is assumed constant at all temperatures.
```

Reply:
`If a scalar value is passed in, the ionization fraction is assumed constant at all temperatures.`

I hadn't added this functionality.  Does your suggestion above fix this?